src/components/views/rooms/SendMessageComposer.tsx

```diff
@@ -164,6 +164,20 @@ export default class SendMessageComposer extends React.Component<IProps> {
         window.addEventListener("beforeunload", this.saveStoredEditorState);
     }
 
+    public componentDidUpdate(prevProps: IProps): void {
+        const replyToEventChanged = this.props.replyToEvent !== prevProps.replyToEvent;
+        if (replyToEventChanged) {
+            this.model.reset([]);
```

Comment:
This might have the unintended side effect of obliterating a user's message after they typed something and changed which message they were replying to - has this case been tested?

Reply:
It had not! Once again thanks. I've scoped this behaviour to the "reply in thread" composer which then behaves the way I intended